assets/bingo/lang/pl_pl.json

```
@@ -0,0 +1,411 @@
+	"bingo.lobby.credits.tools.nbt_explorer.sign.line4": "Justin Aquadro",
+
+	"bingo.lobby.credits.tools.nbt_studio.sign.line3": "od tryashtar",
+	"bingo.lobby.credits.tools.nbt_studio.sign.line4": "",
+
+	"bingo.lobby.credits.tools.nbt_viewer.sign.line3": "od Misode",
+	"bingo.lobby.credits.tools.nbt_viewer.sign.line4": "",
+
+
+	"bingo.lobby.card_generation.random_card.sign.line1": "",
+	"bingo.lobby.card_generation.random_card.sign.line2": "Wygeneruj",
+	"bingo.lobby.card_generation.random_card.sign.line3": "Losową Kartę",
+	"bingo.lobby.card_generation.random_card.sign.line4": "",
+
+	"bingo.lobby.card_generation.from_seed.sign.line1": "",
+	"bingo.lobby.card_generation.from_seed.sign.line2": "Wygeneruj Kartę",
+	"bingo.lobby.card_generation.from_seed.sign.line3": "Z Ziarna",
+	"bingo.lobby.card_generation.from_seed.sign.line4": "",
+
+	"bingo.lobby.card_generation.start_game.sign.line1": "",
+	"bingo.lobby.card_generation.start_game.sign.line2": "Start",
+	"bingo.lobby.card_generation.start_game.sign.line3": "Gry",
+	"bingo.lobby.card_generation.start_game.sign.line4": "",
+
+	"bingo.lobby.card_generation.crafting_table.sign.line1": "Wyszukiwarka",
+	"bingo.lobby.card_generation.crafting_table.sign.line2": "Receptur",
+	"bingo.lobby.card_generation.crafting_table.sign.line3": "Rzemieślni-",
+	"bingo.lobby.card_generation.crafting_table.sign.line4": "czych",
+
+	"bingo.lobby.card_generation.furnace.sign.line1": "Wyszukiwarka",
+	"bingo.lobby.card_generation.furnace.sign.line2": "Receptur",
+	"bingo.lobby.card_generation.furnace.sign.line3": "Przetapiania",
+	"bingo.lobby.card_generation.furnace.sign.line4": "",
+
+	"bingo.lobby.card_generation.from_seed.instructions": "%s aby dostać polecenie na ustawienie ziarna. Wprowadź liczbowe ziarno na końcu polecenia.",
+	"bingo.lobby.card_generation.from_seed.instructions.link": "Kliknij tutaj",
+	"bingo.lobby.card_generation.from_seed.explanation": "Ziarno określa jakie przedmioty zostaną wygenerowane na karcie bingo i gdzie się zespawnujesz. Może to być wykorzystane np. w zawodach, w których wszyscy uczestnicy używają tego samego ziarna.",
+
+	"bingo.lobby.card_generation.start_game.already_in_progress": "Nie możesz rozpocząć nowej gry bez wygenerowania nowej karty.",
+	"bingo.lobby.card_generation.start_game.no_card": "Nie możesz rozpocząć gry bez wygenerowania Karty Bingo.",
+
+	"bingo.lobby.settings.player.tutorial.sign.line1": "Gracz",
+	"bingo.lobby.settings.player.tutorial.sign.line2": "",
+	"bingo.lobby.settings.player.tutorial.sign.line3": "Wyjaśnienie",
+	"bingo.lobby.settings.player.tutorial.sign.line4": "jak to działa",
+
+	"bingo.lobby.settings.player.tutorial.part1": "Te ustawienia umożliwiają ustawić z jakimi przedmiotami i efektami gracz zaczyna grę.\nWszystkie efekty mikstur są trwałe.",
+	"bingo.lobby.settings.player.tutorial.part2": "Istnieje już kilka gotowych konfiguracji, które możesz wykorzystać klikając na tabliczkę “Wybór/Podgląd Konfiguracji” po prawej.",
```

```suggestion
	"bingo.lobby.settings.player.tutorial.part2": "Istnieje już kilka gotowych konfiguracji, które możesz wykorzystać klikając na tabliczkę „Wybór/Podgląd Konfiguracji” po prawej.",
```
Quotation mark styling